facefusion/face_masker.py

```diff
@@ -159,3 +159,8 @@ def create_mouth_mask(face_landmark_68 : FaceLandmark68) -> Mask:
 	mouth_mask = cv2.erode(mouth_mask.clip(0, 1), numpy.ones((21, 3)))
 	mouth_mask = cv2.GaussianBlur(mouth_mask, (0, 0), sigmaX = 1, sigmaY = 15)
 	return mouth_mask
+
+
+def create_face_mask(crop_vision_frame : VisionFrame) -> Mask:
```

Comment:
I wonder why this was not possible for the mouth mask :-)

Reply:
doing with landmarker is cheaper